docs/docs/flink-connector.md

```diff
@@ -61,8 +64,7 @@ CREATE TABLE flink_table (
 ```
 
 !!! info
-    The underlying catalog database (`hive_db` in the above example) will be created automatically if it does not exist when writing records into the Flink table.
-
+The underlying catalog database (`hive_db` in the above example) will be created automatically if it does not exist when writing records into the Flink table.
```

Comment:
nit: we should spot check some of these changes. i think the white space here is required for the `!!! info` block. Otherwise it wont be rendered correctly

Reply:
Yes, I'm checking page by page and find other issues as well.